benchmark/operations/create_container.py

```diff
@@ -0,0 +1,319 @@
+# Copyright 2016 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Operation to create a container.
+"""
+
+from functools import partial
+from itertools import repeat
+import random
+from uuid import UUID, uuid4
+
+from pyrsistent import PClass, field
+from zope.interface import implementer
+
+from flocker.apiclient import MountedDataset
+from flocker.common import gather_deferreds, loop_until
+from flocker.control import DockerImage
+
+from .._interfaces import IProbe, IOperation
+
+
+class EmptyClusterError(Exception):
+    """
+    Exception indicating that the cluster contains no nodes.
+    """
+
+
+def create_dataset(
```

Comment:
:question: I'd consider either renaming this to `_create_dataset`, or adding some tests that exercise it directly. Likewise for some of the other top-level functions.